tensorflow_model_analysis/notebook/jupyter/js/lib/labplugin.js

```diff
@@ -0,0 +1,15 @@
+var plugin = require('./index');
+var base = require('@jupyter-widgets/base');
+
+module.exports = {
+  id: 'tensorflow_model_analysis',
+  requires: [base.IJupyterWidgetRegistry],
+  activate: function(app, widgets) {
+      widgets.registerWidget({
+          name: 'tensorflow_model_analysis',
+          version: plugin.version,
+          exports: plugin
```

Comment:
This exposes all the JS models and views in widgets.js. https://github.com/tensorflow/model-analysis/blob/a96a9665e57a12cb4d3ad1f366dc347347e3a96e/tensorflow_model_analysis/notebook/jupyter/js/lib/widget.js#L238